include/libdnf5/conf/vars.hpp

```diff
@@ -88,6 +88,12 @@ struct Vars {
 
     static std::unique_ptr<std::string> detect_release(const BaseWeakPtr & base, const std::string & install_root_path);
 
+    /// @brief Split releasever on the first "." into its "major" and "minor" components
+    ///
+    /// @param releasever A releasever string, possibly containing a "."
+    /// @return releasever_major, releasever_minor
+    static std::tuple<std::string, std::string> split_releasever(const std::string & releasever);
```

Comment:
Be honest I would not prefer to add the method to API and make it private. If we will add support of releasever_patch value, the method will be obsoleted.

Reply:
Done, makes sense. An API user could always read the value of `releasever_major`, `releasever_minor` after setting `releasever`.